### PR TITLE
feat(google-maps): Update map size in updateRender method if it has changed

### PR DESCRIPTION
**Pull request by AE1NS**
In this PR I added the feature to update the map size when the html wrapper container changes its size. Its implemented with the help of a ResizeObserver which then calls the native `updateRender` methods.

Closes https://github.com/ionic-team/capacitor-plugins/issues/1594

### CODE REVIEW

**Comment by AE1NS** [comment]
I extended the PR to fix https://github.com/ionic-team/capacitor-plugins/issues/997 as its also fixed with the help of the ResizeObserver I implemented here.

Its an adjustment where the map wasnt displayed, i.e. after an ionic back navigation. With the help of the ResizeObserver, a 'hidden' or 'show' state of the element can be tracked and a native method in iOS can fix the issue. Unfortunately I had to add a setTimeout of 700ms to get it work, which seems to be related to the page transition, but Im not sure, otherwise the map would still be not visible.

**Review by theproducer — APPROVED** [review]

**Review by markemer — APPROVED** [review]

File: google-maps/package.json [hunk 1]
```diff
@@ -62,6 +62,8 @@
     "@capacitor/ios": "^5.0.0",
     "@ionic/eslint-config": "^0.3.0",
     "@ionic/prettier-config": "^1.0.1",
+    "@ionic/swiftlint-config": "^1.1.2",
```
Comment (IT-MikeS) [inline google-maps/package.json:65]: This is not needed as the swift lint config was changed to use just references from the `swiftlint.config.js` file.

**Review by IT-MikeS — CHANGES_REQUESTED** [review]

File: google-maps/package.json (hunk 1 above)
Comment (AE1NS) [reply google-maps/package.json:65]: Did something go wrong with the last merge commit @theproducer? I didnt touched this line.

Comment (theproducer) [reply google-maps/package.json:65]: Possibly.  I'll remove it.

**Review by IT-MikeS — APPROVED** [review]

**Pull request merged**